backend/molgenis-emx2-rdf/src/main/java/org/molgenis/emx2/rdf/RDFService.java

```diff
@@ -148,24 +153,29 @@ public void describeAsRDF(
       final Schema... schemas) {
     try {
       final ModelBuilder builder = new ModelBuilder();
-      builder.setNamespace("rdf", NAMESPACE_RDF);
```

Comment:
What happens if same namespace is defined twice?
Could we simply keep this and then add the RDF setting to override?

Reply:
I'll make sure to add tests for:
* Identical copies -> should be non-issue
* Same namespace with different prefix -> no clue tbh
* Same prefix different namespace -> probably replaces and last-added is used?